content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,278 @@
+
+
+9. Navigate to the left **Main Menu** and select **Profiles**. 
+
+
+10. Click on the specific cluster profile to view its details. 
+
+
+11. Examine the cluster details page URL. The cluster details page URL follows the `[Palette-URL]/projects/[PROJECT-ID]/profiles/cluster/[CLUSTER-PROFILE-ID]` syntax. The cluster details page URL has your project ID and the cluster profile ID. For example, the screenshot below highlights the project ID and the cluster profile ID in a cluster details page URL. 
+
+  ![A screenshot highlighting the cluster profile ID and project ID in the URL of the cluster details page.](/clusters_edge_edge-forge-workflow_build-images_build-artifacts_url.png)
+
+
+
+12. Copy the cluster profile ID from the cluster details page URL for the next step. 
+
+
+13. Switch back to your development environment, and set the project ID as an environment variable by issuing the following command. Replace the `[USE-YOUR-PROJECT-ID_HERE]` placeholder with your project ID.
+<br />
+
+  ```bash
+  export PROJECT_ID=[USE-YOUR-PROJECT-ID_HERE]
+  ```
+
+
+14. Set the cluster profile ID as an environment variable using the following command. Replace the `[USE-YOUR-PROFILE-ID_HERE]` placeholder with your cluster profile ID. The Palette Edge CLI uses the cluster profile ID to reference the correct cluster profile and download all its software dependencies. 
+<br />
+
+  ```bash
+  export PROFILE_ID=[USE-YOUR-PROFILE-ID_HERE]
+  ```
+
+
+15. Use the command below to create the content bundle. The command uses the following flags:
+
+  |**Command Flag**|**Value**|
+  |---|---|
+  |`--api-key`|Spectro Cloud API key|
+  |`--project-id`|Palette project ID|
+  |`--cluster-profile-ids`|Cluster profile IDs. If you want to include multiple cluster profiles in the content bundle, add multiple cluster profile IDs separated by a comma.|
+  |`--palette-endpoint`|Palette API endpoint. The default Palette API endpoint is `api.spectrocloud.com`|
+  |`--outfile`|Path to write the final content bundle. |
+  You can issue `palette-edge build --help` to know about other available flags. 
+<br />
+
+  ```bash
+  palette-edge build --api-key $API_KEY \
+    --project-id $PROJECT_ID \
+    --cluster-profile-ids $PROFILE_ID \
+    --palette-endpoint api.spectrocloud.com \
+    --outfile content
+  ```
+  
+
+
+16. Use the command below to list all files in the current directory to verify that you created the content bundle successfully. The content bundle will have the following naming convention, `content-[randon-string]`, for example, **content-8e61a9e5**. 
+<br />
+
+  ```bash
+  ls -al
+  ```
+
+
+17. List the files in the content bundle folder using the following command. Replace the `content-[randon-string]` with the content bundle in your current directory. The output will display the compressed files for the core and the app content. 
+<br />
+
+  ```bash
+  ls -l content-[randon-string]
```

Comment:
```suggestion
  ls -la content-*/
```
We can use `ls -la content-*/` . This way, the user only sees folders that start with `content-` and their content.